auth_admin_passkey/__manifest__.py

```diff
@@ -4,26 +4,21 @@
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
-    'name': 'Authentification - Admin Passkey',
-    'version': '10.0.1.0.2',
-    'category': 'base',
-    'author': "GRAP,Odoo Community Association (OCA)",
-    'website': 'http://www.grap.coop',
-    'license': 'AGPL-3',
-    'depends': [
-        'mail',
-    ],
-    'data': [
-        'data/ir_config_parameter.xml',
-        'views/res_config_view.xml',
-    ],
-    'demo': [],
-    'js': [],
-    'css': [],
-    'qweb': [],
-    'images': [],
-    'post_load': '',
-    'application': False,
-    'installable': True,
-    'auto_install': False,
+    "name": "Authentification - Admin Passkey",
+    "version": "10.0.1.0.2",
+    "category": "base",
+    "author": "GRAP,Odoo Community Association (OCA)",
+    "website": "http://www.grap.coop",
+    "license": "AGPL-3",
+    "depends": ["mail"],
+    "data": ["data/ir_config_parameter.xml", "views/res_config_view.xml"],
+    "demo": [],
```

Comment:
Empty keys should be deleted. Actually you can remove application and auto_install keys as well.

Reply:
I've removed those, and ran 'isort' as well over the first commit.